libs/constants/poolLists.ts

```diff
@@ -0,0 +1,235 @@
+import { AvailableNetwork } from '@context/Web3Context/Web3Context.Config';
+import { StaticPoolInfo } from 'libs/types/General';
+import BigNumber from 'bignumber.js';
+import { ARBITRUM, ARBITRUM_RINKEBY } from '.';
+
+const ONE_HOUR = new BigNumber(3600); // seconds
+const FIVE_MINUTES = new BigNumber(300); // seconds
+
+export const poolList: Record<AvailableNetwork, StaticPoolInfo[]> = {
+    [ARBITRUM]: [
+        {
+            name: '1-BTC/USD',
+            address: '0x146808f54DB24Be2902CA9f595AD8f27f56B2E76',
+            leverage: 1,
+            updateInterval: ONE_HOUR,
+            frontRunningInterval: FIVE_MINUTES,
+            keeper: '0x759E817F0C40B11C775d1071d466B5ff5c6ce28e',
+            longToken: {
+                name: '1-LONG-BTC/USD',
+                address: '0x1616bF7bbd60E57f961E83A602B6b9Abb6E6CAFc',
+                symbol: '1L-BTC/USD',
+                decimals: 6,
```

Comment:
this should be 18 for all pool tokens i believe?

Reply:
yep